appeals/web/src/server/appeals/appeal-details/interested-party-comments/add-ip-comment/add-ip-comment.service.js

```diff
@@ -16,3 +16,33 @@ export const getAttachmentsFolder = async (apiClient, appealId) => {
 
 	return folders[0];
 };
+
+/**
+ * @param {{ firstName: string, lastName: string, emailAddress: string, addressLine1: string, addressLine2: string, town: string, county: string, postCode: string, redactionStatus: boolean, 'date-day': string, 'date-month': string, 'date-year': string }} comment
```

Comment:
Ideally there would be a typedef in `web/src/server/appeals/appeal-details/interested-party-comments/interested-party-comments.types.d.ts` for the `comment` type - even if it's not used elsewhere, it's pretty verbose to define inline

Reply:
try `/** @typedef {import("#appeals/appeal-details/interested-party-comments/interested-party-comments.types.js").Representation} Representation */
`

Reply:
looks like you made an interface for this but forgor to use it?